{{ cookiecutter.__project_name_kebab }}/pyproject.toml

```diff
@@ -49,7 +49,8 @@ dependencies = [
 ]
 [project.optional-dependencies]
 testing = [
-    "dj-database-url==1.2.0"
+    "dj-database-url==1.2.0",
+    "pre-commit>=2.21.0,<3.0"
```

Comment:
I've pinned to pre-commit 2.x because pre-commit 3 drops compatibility with python 3.7 and this project still includes 3.7 in the test matrix/classifiers. Once we drop it, we can bump this to pre-commit 3

Reply:
👍🏼  #50